design/20190708.certificate-request-crd.md

```diff
@@ -397,8 +398,13 @@ minimal as possible in that the single goal of them is to enable its owning
 `CertificateRequest` has been observed, the general flow is as follows:
 
 - Check the group belongs to the owning `Issuer`, exit if not.
-- Check if `CertificateRequest` is in a failed state, exit if true. TODO: more
-  tightly define what a 'failed state' exactly is.
+- Check if `CertificateRequest` is in a failed state.
+  An controller may choose to add additional conditions to a failed `CertificateRequest`, but must not attempt to issue a certificate.
+  Currently failed states are:
```

Comment:
Could `Currently` be changed to specify a version (`As of cert-manager version X`)?

It'll make it easier for future readers to have a sense of how likely it is for the list to be "current" when they read it.

Reply:
If more conditions get added then this doc should be updated. However, there is a warning at the top that this doc may be out of date